cardano-cli/src/Cardano/CLI/EraBased/Query/Run.hs

```diff
@@ -1101,11 +1102,22 @@ writeStakeAddressInfo
         )
         merged
 
-    friendlyDRep :: L.DRep -> Text
+    friendlyStake :: PoolId -> Aeson.Value
+    friendlyStake poolId@(StakePoolKeyHash (C.KeyHash hash)) =
+      Aeson.object
+        [ "stakePoolBech32" .= serialiseToBech32 poolId
+        , "stakePoolHex" .= hashToTextAsHex hash
+        ]
+
+    friendlyDRep :: L.DRep -> Aeson.Value
     friendlyDRep L.DRepAlwaysAbstain = "alwaysAbstain"
     friendlyDRep L.DRepAlwaysNoConfidence = "alwaysNoConfidence"
-    friendlyDRep (L.DRepCredential cred) =
-      L.credToText cred -- this will pring "keyHash-..." or "scriptHash-...", depending on the type of credential
+    friendlyDRep (L.DRepCredential (L.ScriptHashObj (C.ScriptHash hash))) = Aeson.String $ "scriptHash-" <> hashToTextAsHex hash
+    friendlyDRep (L.DRepCredential (L.KeyHashObj kh@(C.KeyHash cred))) =
```

Comment:
This feels inconsistent: script hash will be printed as `scriptHash-xxxx` where xxxx is a hash, but key hash will be rendered just as hash. Why not encode it the same way?

Also the same remark as above, I suggest using `Using{Bech32,RawBytesHex}` newtype wrappers.